theme/custom.scss

```diff
@@ -500,3 +500,30 @@ a {
 .overflow-table {
   overflow-x: scroll;
 }
+
+/* 
+  Bottom aligned header rows 
+*/
+.bottom-aligned {
+  display: inline;
+  vertical-align: bottom;
+}
+
+/*
+  Fully span the row width for segmented columns
+*/
+.full-width-span {
+  flex: 1 1 100%;
+}
+/*
+  Generic display flex for when necessary
+*/
+.display-flex {
+  display: flex;
+}
```

Comment:
Delete. This is [already in USWDS](https://designsystem.digital.gov/utilities/display/).

```suggestion
```